Project21/Project21/comp.c

```diff
@@ -0,0 +1,49 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// complex.c
+// Functions performing on the complex numbers
+// Naveenkumar s
+// ------------------------------------------------------------------------------------------------
+
+#include"comp.h"
+#include<math.h>
+#include<stdio.h>
+
+Complex Op_Add (Complex c1, Complex c2) {
+   Complex output = { 0.0,0.0 };
+   output.Realpart = c1.Realpart + c2.Realpart;
+   output.Imagpart = c1.Imagpart + c2.Imagpart;
+   return output;
```

Comment:
Initial values for variables need not be Zero. You can directly use your desired values. Similarly fix everywhere.
```suggestion
   Complex output = {  c1.Realpart + c2.Realpart, c1.Imagpart + c2.Imagpart };
   return output;
```